### ISSUE TITLE
[CoE Starter Kit - BUG] ISSUE TITLE

### ISSUE THREAD

**Issue by declawson** [microsoft/coe-starter-kit#3742]
### Describe the issue

Action 'Maker_Check_-_Edit_Connector' failed for 'default' and 'COE' envs for Admin | Sync Template v3 (Custom Connectors)

### Expected Behavior

At the connection action getting a 'No Repsonse' error.

### What solution are you experiencing the issue with?

Core

### What solution version are you using?

3.44

### What app or flow are you having the issue with?

Admin | Sync Template v3 (Custom Connectors)

**Comment by Jenefer-Monroe** [comment]
Hello can you please share a screen shot of what you are referencing?

**Comment by Jenefer-Monroe** [comment]
closing out as no further action for starter kit team

**Comment by declawson** [comment]
I see this ticket was closed but I didn’t receive any response about what the issue might be or how to resolve.

[https://webmail.atcc.org/owa/auth/image001.jpg]
Debbie Clawson
Senior IT Architect

10801 University Boulevard
Manassas, VA 20110-2209
Tel: (703) 365-2700 ext. 2820
Fax: (703) 365-2701
Email: ***@***.******@***.***>

**Comment by Jenefer-Monroe** [comment]
Hello. The last action item was on you to please share a screenshot so that we could understand the issue. 
If you would like to continue the investigation please do that and we can re-open. Note that you need to log in to GitHub to share screenshots, sharing in email does not send the image to us.

**Comment by declawson** [comment]
I provided screenshots 9/23, see attached email.

[https://webmail.atcc.org/owa/auth/image001.jpg]
Debbie Clawson
Senior IT Architect

10801 University Boulevard
Manassas, VA 20110-2209
Tel: (703) 365-2700 ext. 2820
Fax: (703) 365-2701
Email: ***@***.******@***.***>

**Comment by declawson** [comment]
I was not aware that you would not receive the screenshares when sending by email.  I will login into GitHub and provide them.

[https://webmail.atcc.org/owa/auth/image001.jpg]
Debbie Clawson
Senior IT Architect

10801 University Boulevard
Manassas, VA 20110-2209
Tel: (703) 365-2700 ext. 2820
Fax: (703) 365-2701
Email: ***@***.******@***.***>


From: Clawson, Debbie
Sent: Tuesday, October 4, 2022 10:38 AM
To: microsoft/coe-starter-kit ***@***.***>; microsoft/coe-starter-kit ***@***.***>
Cc: Author ***@***.***>
Subject: RE: [microsoft/coe-starter-kit] [CoE Starter Kit - BUG] ISSUE TITLE (Issue #3742)

I provided screenshots 9/23, see attached email.

[https://webmail.atcc.org/owa/auth/image001.jpg]
Debbie Clawson
Senior IT Architect

10801 University Boulevard
Manassas, VA 20110-2209
Tel: (703) 365-2700 ext. 2820
Fax: (703) 365-2701
Email: ***@***.******@***.***>

**Comment by Jenefer-Monroe** [comment]
Hello you cannot share pictures in email. We do not work through emails for these and then also technically GitHub does not share those anyway. You need to log in to GitHub, open this bug, and share a screenshot.

**Comment by declawson** [comment]
Here are the screenshots:   ‘Admin | Sync Template v3 (Custom Connectors) consistently fails in my ‘Default’ and ‘COE’ environments.  It runs on the other envs.
![28-day run history](https://user-images.githubusercontent.com/74382505/193898249-a65bd267-fe2e-454c-9b18-4354079c6137.png)
![Flow run failed](https://user-images.githubusercontent.com/74382505/193898324-ee8c84a4-3a98-408d-afac-41a17fe65932.png)
![Get Custom Connectors](https://user-images.githubusercontent.com/74382505/193898400-cb176a6a-fb23-4a74-a1bb-79e77d90e686.png)

**Comment by Jenefer-Monroe** [comment]
Please open the **_See if already in CoE_** action adn keep going down until you find where it terminates.

**Comment by declawson** [comment]
![See if already in COE](https://user-images.githubusercontent.com/74382505/193903842-32827d19-8f0a-4c21-9cca-3d9859d767e9.png)
![Maker Check - Edit Connector](https://user-images.githubusercontent.com/74382505/193903930-13f791f1-305a-4b9d-976c-48f18d908f06.png)
![Check if Orphan](https://user-images.githubusercontent.com/74382505/193904108-1b8cc800-9a76-4de9-ae78-6a04ec124195.png)

**Comment by Jenefer-Monroe** [comment]
Thank you, it looks like you dont have the **_HELPER – Maker Check_** helper flow turned on. 
Please review all the [inventory flows](https://learn.microsoft.com/power-platform/guidance/coe/setup-core-components) to be sure they are all turned on.  Note that there are 3 sections to look through
1) [Helper flows](https://learn.microsoft.com/power-platform/guidance/coe/setup-core-components#turn-on-child-flows)
2) [Setup flows](https://learn.microsoft.com/power-platform/guidance/coe/setup-core-components#turn-on-setup-flows)
3) [Inventory flows](https://learn.microsoft.com/power-platform/guidance/coe/setup-core-components#turn-on-inventory-flows)

**Comment by Jenefer-Monroe** [comment]
I've never seen that happen before. Do you have an unmanaged layer on that helper flow?

**Comment by declawson** [comment]
No

![image](https://user-images.githubusercontent.com/74382505/194118249-d8b6a146-756f-44f3-b173-9f56055880f9.png)

**Comment by Jenefer-Monroe** [comment]
Please open the flow so we can look at it in the editor

**Comment by Jenefer-Monroe** [comment]
ok it looks like your Custom Connector sync flow isnt sending in the Recheck variable. This is what you should see there and you only see the GUID
![image](https://user-images.githubusercontent.com/89584709/194294957-852c2267-3886-438c-8514-477350860529.png)

Please check to see if you have unamanged layers on **_Admin | Sync Template v3 (Custom Connectors)_**

**Comment by declawson** [comment]
Yes, can I assume you want me to remove the unmanaged layer.  That must have been added during the upgrade because I haven't started adding unmanaged layers except for env vars and connections.

https://atcccloud.sharepoint.com/teams/ProcurementServiceContracts/Lists/Service%20Contracts/Active%20Contracts.aspx

**Comment by declawson** [comment]
All runs succeeded last night.

**Comment by Jenefer-Monroe** [comment]
Wonderful news, thanks for letting me know!

**Comment by Jenefer-Monroe** [comment]
You will need to post new issues for additional problems: https://github.com/microsoft/coe-starter-kit/issues/new/choose

**Comment by Jenefer-Monroe** [comment]
and again, screenshots cannot be shared by email.
